plugins/auth0/pkg/fetch/fetch.go

```diff
@@ -136,6 +110,45 @@ func (f *Fetcher) fetchUsers(ctx context.Context, outputWriter *js.JSONArrayWrit
 	return nil
 }
 
+// return a object map to output or a boolean to skip current user.
+func (f *Fetcher) buildOutputObjects(ctx context.Context, user *management.User, errorWriter io.Writer) (map[string]any, bool) {
```

Comment:
It makes no sense for this function to take an `errorWriter`. It's not taking an `outputWriter` so why should it know about writing errors. It should simply return any error it encounters and let the caller decide what to do about it.

Reply:
And it doesn't need to return a bool either, if there are no objects to write it can return an empty map.